### ISSUE TITLE
Op type not registered 'NormalizeUTF8' in binary running when reload model #51080

### ISSUE THREAD

**Issue by jayaBalaR** [tensorflow/tensorflow#52240]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Google Colaboratory
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: No
- TensorFlow installed from (source or binary): 2.6
- TensorFlow version (use command below):
- Python version: Python 3.7.12
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory: GPU

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with:
1. TF 1.0: `python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"`
2. TF 2.0: `python -c "import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**
Actually , I am trying to convert the saved_model to tensorflowlite model and while using this code
import tensorflow as tf

# Convert the model
converter = tf.lite.TFLiteConverter.from_saved_model('./translator') # path to the SavedModel directory
tflite_model = converter.convert()

# Save the model.
with open('model.tflite', 'wb') as f:
  f.write(tflite_model)

I am getting this error
![image](https://user-images.githubusercontent.com/21074002/135761699-070a32cf-7387-48bf-a60a-cad371eb855a.png)

**Comment by tilakrayal** [comment]
@jayaBalaR ,
Can you please take a look at this [comment](https://github.com/tensorflow/tensorflow/issues/49968#issuecomment-852677186) from the issue with similar error.It helps.Thanks!

**Comment by jayaBalaR** [comment]
Yes NormalizeUTF8 's corresponding library seems to be missing while packaging . I am not sure where to include this issue for tflite. Please reassign to that github issues. thanks

**Comment by sanatmpa1** [comment]
@jayaBalaR,

Can you provide us a simple reproducible code snippet or colab gist of the issue to expedite the trouble-shooting process? Thanks!

**Comment by jayaBalaR** [comment]
@sanatmpa1 
I have used the code in this https://www.tensorflow.org/lite/convert/ 
# Convert the model
converter = tf.lite.TFLiteConverter.from_saved_model(saved_model_dir) # path to the SavedModel directory
tflite_model = converter.convert()

# Save the model.
with open('model.tflite', 'wb') as f:
  f.write(tflite_model)

to convert my saved_model to tflite model, and it fails in the first line.
I have created the saved_model using
tf.saved_model.save(translator, 'translator',
                    signatures={'serving_default': translator.tf_translate})

**Comment by sanatmpa1** [comment]
@jayaBalaR,

I tried this example in the [link](https://www.tensorflow.org/lite/convert/#convert_a_keras_model_) you provided with a slight modification, and I don't see any errors. You can take a look at the [gist here](https://colab.research.google.com/gist/sanatmpa1/7c153b94125e63066cfcdcf54cfd03e3/52240.ipynb). If this is not the right code, Please point me to the right example, or provide a colab gist of standalone code to reproduce the error. Thanks!
